bot/resources/tags/in-place.md

```diff
@@ -0,0 +1,19 @@
+**Out of Place** and **In Place**
+
+In programming, there are two types of operations: "out of place" operations create a new object, leaving the original object unchanged. "in place" operations modify the original object, without creating a new one. These operations return None explicitly.
+
+A common example of these different concepts is seen in the use of the methods `list.sort()` and `sorted(...)`. Using `list.sort()` and attempting to access an element of the list will result in an error. 
+
+```py
+# WRONG:
+
+unsorted_list = [3, 1, 2]
+sorted_list = unsorted_list.sort() # This will be None
```

Comment:
[PEP 257](https://peps.python.org/pep-0008/#inline-comments) 
> An inline comment is a comment on the same line as a statement. Inline comments should be separated by at least two spaces from the statement. They should start with a # and a single space.

Should be changed to
```python
sorted_list = unsorted_list.sort()  # This will be None
```